Android-Sample/README.md

```diff
@@ -16,217 +20,108 @@ The sample app only subscribes to the test stream. It does not subscribe to othe
-
-1. In Android Studio, select the File > Import Project command. Navigate to the root directory of
-   this project, select the build.gradle file, and then click the OK button. The project opens in a
-   new window.
-
-   The Java code for the application is the MainActivity class in the
-   com.tokbox.qualitystats.sample package.
-
-2. Sync Gradle
-
-3. In the com.opentok.qualitystats.sample.MainAcdtivity.java class, set the following properties
-   to a test OpenTok session ID, token, and API key:
+## Usage
 
-   ```
-   private static final String SESSION_ID = "";
-   private static final String TOKEN = "";
-   private static final String APIKEY = "";
-   ```
+1. **Import the Class**
 
-   **Important:** You must use a unique session, with its own session ID, for the network test. This
-   must be a different session than the one you will use to share audio-video streams between
-   clients. Do not publish more than one stream to the test session.
+    Make sure you've imported the `NetworkQualityTest` class into your project.
 
-   The app requires that each session uses the routed media mode -- one that uses
-   the [OpenTok Media Router](https://tokbox.com/developer/guides/create-session/#media-mode).
-   A routed session is required to get statistics for the stream published by the local client.
+    ```java
+    import com.opentok.qualitystats.sample.NetworkQualityTest;
+    ```
 
-   You can get your API key as well as a test session ID and token at the
-   [OpenTok dashboard](https://dashboard.tokbox.com/). However, in a shipping application, use
-   one of the [OpenTok server SDKs](https://tokbox.com/developer/sdks/server/) to generate a
-   session ID and token.
+2. **Instantiate the Class**
 
-4. Debug the project on a supported device.
+    Create an instance of `NetworkQualityTest` by providing the required parameters: the current activity, a configuration object, and a callback listener for receiving the test results.
 
-   For a list of supported devices, see the "Developer and client requirements"
-   on [this page](https://tokbox.com/developer/sdks/android/).
+    ```java
+    NetworkQualityTest networkQualityTest = new NetworkQualityTest(
+        this,  // Current activity
+        networkQualityTestConfig,  // Configuration object
+        networkQualityTestCallbackListener  // Callback listener
+    );
+    ```
 
- The app uses a test stream and session to determine the client's ability to publish a stream
- that has audio and video. At the end of the test it reports one of the following:
+3. **NetworkQualityTestConfig**
 
-   * You're all set -- your client can publish an audio-video stream.
+    The `NetworkQualityTestConfig` class is used to configure the parameters of the network quality test. It provides the following constructor:
 
-   * Voice-only -- Your bandwidth is too low for video.
+    You can configure the following parameters:
+    
+    - `sessionId`: The session ID of your OpenTok session.
```

Comment:
Mention that this should be a [routed session](https://tokbox.com/developer/guides/create-session/#media-mode) (right?) And mention that this session should only be used for the test (not for communications).